api/accesslog.proto

```diff
@@ -0,0 +1,138 @@
+syntax = "proto3";
+
+package envoy.api.v2;
+
+import "api/base.proto";
+
+import "google/protobuf/duration.proto";
+import "google/protobuf/struct.proto";
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/wrappers.proto";
+
+message EnvoyAccessLog {
+  // The request method, ie, "GET", "HEAD", "POST"
+  string request_method = 1;
+
+  // The time we started servicing this request
+  google.protobuf.Timestamp start_time = 2;
```

Comment:
Since the end time can be easily calculated by adding the response duration to the start time I don't think I'd make it a separate field.